posthog/api/capture.py

```diff
@@ -106,10 +107,14 @@ def get_event(request):
     if not token:
         return cors_response(request, JsonResponse({'code': 'validation', 'message': "No api_key set. You can find your API key in the /setup page in posthog"}, status=400))
 
-    try:
-        team = Team.objects.only('pk').get(api_token=token)
-    except Team.DoesNotExist:
-        return cors_response(request, JsonResponse({'code': 'validation', 'message': "API key is incorrect. You can find your API key in the /setup page in PostHog."}, status=400))
+    team_id = TEAM_ID_CACHE.get(token)
+    if not team_id:
+        try:
+            team_id = Team.objects.only('pk').get(api_token=token).pk
+            if team_id:
+                TEAM_ID_CACHE[token] = team_id
```

Comment:
does this cache ever get flushed?

Reply:
It does when the server gets restarted. However the token is fairly static, I don't think there'd be many situations you'd change it